src/misc_gui.cpp

```diff
@@ -961,21 +975,63 @@ struct QueryStringWindow : public Window
 			case WID_QS_CANCEL:
 				this->Close();
 				break;
+
+			case WID_QS_MOVE: // move
+				this->last_user_action = widget;
+
+				if (Station::IsExpected(Station::Get(this->parent->window_number))) { /* station */
```

Comment:
Inline comments use `//`, also applies to other functions in this file.

```suggestion
				if (Station::IsExpected(Station::Get(this->parent->window_number))) { // This is a station.
```